packages/x-date-pickers/src/internals/hooks/usePickerState.ts

```diff
@@ -21,14 +21,42 @@ export interface PickerStateValueManager<TInputValue, TDateValue> {
 
 export type PickerSelectionState = 'partial' | 'shallow' | 'finish';
 
-interface Draftable<T> {
-  committed: T;
+interface DateState<T> {
+  /**
+   * Date internally used on the picker and displayed in the input.
+   * It is updates whenever the user validates a step.
+   */
   draft: T;
+
+  /**
+   * Last full date provided by the user
+   * Is not updated when validating a step of a multistep picker (e.g. validating the date of a date time picker)
+   */
+  committed: T;
+
+  /**
+   * Date that will be used if the pickers tries to reset its value
+   */
+  accepted: T;
 }
 
-interface DraftAction<DraftValue> {
-  type: 'update' | 'reset';
-  payload: DraftValue;
+type DateStateActionType =
+  /**
+   * Set the draft, committed and accepted dates to the action value
+   */
+  | 'accept'
+  /**
+   * Set the draft and committed date to the action value
+   */
+  | 'commit'
+  /**
+   * Set the draft date to the action value
+   */
+  | 'draft';
```

Comment:
If I correctly understand the pipeline, the only situation in which the difference between `'commit'` and `'accept'` is relevant is when `shouldCloseOnSelect===false`.

In this specific case, you need to keep track of the value before opening the input to do the reset, and this value is saved in `accepted` property.

If that's correct, what about renaming `accepted` with something like `initialValue`, `valueBeforeEditing`, `resetTraget`, or anything that does not refer to validation

Reply:
Yes
I am very unsure of the use case and I am gathering all the issues related for this problem to make sure I am not breaking stuff
But the idea is what you described

I don't like the "initialValue" name because for it's not initial, when you click the "Accept" it updates this value.
The best is probably "resetTarget"